### PR TITLE
PTFM-18184 dx find doesn't shows help option for apps

### PR DESCRIPTION
**Pull request by ivoloshanenko**
https://dnanexus.jira.com/browse/PTFM-18184

### CODE REVIEW

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -4585,7 +4585,11 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 
 parser_find_org = subparsers_find.add_parser(
     "org",
-    help=fill("List entities within a specific org.") + "\n\n\t" + fill('"dx find org members" lists members in the specified org') + "\n\n\t" + fill('"dx find org projects" lists projects billed to the specified org') + "\n\n" + fill('Please execute "dx find org -h" for more information.'),
+    help=fill("List entities within a specific org.") + "\n\n\t" +
+         fill('"dx find org members" lists members in the specified org') + "\n\n\t" +
+         fill('"dx find org projects" lists projects billed to the specified org') + "\n\n\t" +
+         fill('"dx find org apps" list apps billed to the specified org') + "\n\n" +
```
Comment (ptn24) [inline src/python/dxpy/scripts/dx.py]: "list apps" -> "lists apps".

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -4585,7 +4585,11 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 
 parser_find_org = subparsers_find.add_parser(
     "org",
-    help=fill("List entities within a specific org.") + "\n\n\t" + fill('"dx find org members" lists members in the specified org') + "\n\n\t" + fill('"dx find org projects" lists projects billed to the specified org') + "\n\n" + fill('Please execute "dx find org -h" for more information.'),
+    help=fill("List entities within a specific org.") + "\n\n\t" +
```
Comment (ptn24) [inline src/python/dxpy/scripts/dx.py]: Please indent the subsequent lines of this arg assignment.

Comment (ivoloshanenko) [inline src/python/dxpy/scripts/dx.py]: @ptn24 what do you mean? Looks like I already did this.. :(

Comment (ptn24) [inline src/python/dxpy/scripts/dx.py]: Oh, I misread the lines. My bad.

File: src/python/dxpy/scripts/dx.py (hunk 1 above)
Comment (ptn24) [inline src/python/dxpy/scripts/dx.py]: Ping.

Comment (ivoloshanenko) [inline src/python/dxpy/scripts/dx.py]: Will change ASAP ) Tnx) Just waiting for answer question below.

**Pull request merged**